cmd/oras/internal/option/remote.go

```diff
@@ -118,17 +122,29 @@ func (opts *Remote) ApplyFlagsWithPrefix(fs *pflag.FlagSet, prefix, description
 
 // Parse tries to read password with optional cmd prompt.
 func (opts *Remote) Parse() error {
+	if opts.IdentityToken != "" || opts.IdentityTokenFromStdin {
```

Comment:
Two changes need to be made:
1) the output flag name should include `flagPrefix`. e.g.
```bash
oras cp localhost:5000/from:v1 localhost:7000/to:v1 --from-username test --from-identity-token hello
```
Should output
```
Error: --from-username cannot be used with --from-identity-token
```

Reply:
Added `flagPrefix`